devadm/devadm.go

```diff
@@ -46,8 +46,12 @@ type App interface {
 	DeleteDeviceData(ctx context.Context, id model.DeviceID) error
 
 	ProvisionTenant(ctx context.Context, tenant_id string) error
+
+	PreauthorizeDevice(ctx context.Context, authSet model.AuthSet) error
 }
 
+var AuthSetConflictError = errors.New("AuthSetConflictError")
```

Comment:
'device already exists?' 'authset already exists'? 'identity data already submitted?' not sure about the level of detail, but let's make it readable for the user

Reply:
+1 for "device already exists"

Reply:
Fixed